src/components/TheHeader.vue

```diff
@@ -85,12 +141,66 @@ export default {
 </script>
 
 <style scoped lang="scss">
+.v-enter-active,
+.v-leave-active {
+  transition: opacity 0.5s ease;
+}
+
+.v-enter-from,
+.v-leave-to {
+  opacity: 0;
+}
+
 .header {
   width: 100%;
   height: 100px;
   display: flex;
   justify-content: space-between;
   align-items: center;
+  position: relative;
+
+  .mobile-menu {
+    background-color: white;
+    width: 100%;
+    height: 170px;
+    position: fixed;
+    top: 80px;
+    right: 0;
+    z-index: 99;
+    -webkit-box-shadow: 0px 30px 24px 0px rgba(66, 68, 90, 0.32);
+    -moz-box-shadow: 0px 30px 24px 0px rgba(66, 68, 90, 0.32);
```

Comment:
A throwback to today's talk! Box shadow is widely supported, no need for these at all 😄!